api/src/main/java/jakarta/data/metamodel/restrict/Operator.java

```diff
@@ -27,14 +27,23 @@ public enum Operator {
     LIKE,
     NOT_EQUAL,
     NOT_IN,
-    NOT_LIKE;
+    NOT_LIKE,
+    NOT_PREFIXED,
+    NOT_SUBSTRINGED,
+    NOT_SUFFIXED,
+    PREFIXED,
+    SUBSTRINGED,
+    SUFFIXED;
```

Comment:
FWIW I would not add these unless we were also going to add equivalent operations to JDQL.

Reply:
> FWIW I would not add these unless we were also going to add equivalent operations to JDQL.

In JDQL, these are `LEFT` and `RIGHT` (and substring/contains comes from using `LIKE` in a particular way - that one is less direct).  I copied them over from the `Is` annotation to avoid losing track of them.  If this does go in, there will need to be discussion over what the exact list should be.